src/ol/format/mvtformat.js

```diff
@@ -41,7 +41,7 @@ ol.format.MVT = function(opt_options) {
    * @type {ol.proj.Projection}
    */
   this.defaultDataProjection = new ol.proj.Projection({
-    code: 'EPSG:3857',
+    code: '',
```

Comment:
can you give some background on this change? Does it mean pixel space projections will always have an empty string as the code?